l10n_nl_mis_reports/__manifest__.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2020 Hunki Enterprises BV
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Dutch MIS Builder templates",
+    "summary": "Profit & Loss / Balance sheet for the Netherlands",
+    "version": "12.0.1.0.0",
+    "license": "AGPL-3",
```

Comment:
Please add category so it shows up corretly in odoo appstore and backend applist.
"category": "Localization",
or
"category": "Localization\Netherlands",